hugo/content/contribute.md

```diff
@@ -3,83 +3,261 @@ title: Contribute
 
-# Suggest Changes and Features
+# Development
 
-The issue tracker is also used to track [feature requests, which are marked with the `feature-request` label](https://github.com/mumble-voip/mumble/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request+sort%3Areactions-%2B1-desc).
+## Coding
 
-Please vote with a thumbs up on those you would like to see being fulfilled, describe your use cases or otherwise participate in the discussion on existing tickets. And if you want to request a feature not suggested yet, please create a new feature request ticket possibly with some elaboration and use cases.
+<!-- Todo: 
+- mention Label for important tasks ? (e.g. [gRPC development](https://github.com/mumble-voip/mumble/pull/3947))
 
-# Test new features and developments
+-->
 
-Apart from our stable releases we also provide development snapshots for testing the current, intermediate state of development. You can download them from our [Downloads page]({{< relref "downloads" >}})).
+We are always looking for helping hands willing to contribute to Mumble.
 
-If you find any issues with them please check if the issue has already been reported, add details if you have more, or add a new tracker item if it has not been reported yet.
+Mumble is mostly written in **C++** using the **Qt** library.
+Thus previous knowledge in C++ and Qt is definitely useful, but we also encourage interested people who want to learn it to give it a try. We're happy to help you out, if needed.
 
-# Translation
+Our source code is hosted on GitHub:
 
-Mumble currently (April 2020) supports over 15 different languages (38 if including unfinished translations). As more people get to know and use Mumble, versions in their native languages become more and more important.
+- [Overview of Mumble Projects](https://github.com/mumble-voip/)
 
-We are thankful for any contribution on translating. Please check the {{< wiki "Language Translation" "Language Translation wiki page" />}} for more information.
+- [Main Mumble Repo](https://github.com/mumble-voip/mumble): includes the Server and Client
+  
+- [iOS app](https://github.com/mumble-voip/mumble-iphoneos): unmaintained; help wanted
 
-# Documentation
+### Getting Started
 
-Mumble and all around it needs documentation. Great documentation for a great program.
+The best way to get into Mumble development is to link up with us on IRC ([#mumble-dev on freenode](irc://chat.freenode.net/mumble-dev)).
 
-**Website**: This website has a public hugo source repository at [mumble-www](https://github.com/mumble-voip/mumble-www). You can report [issues](https://github.com/mumble-voip/mumble-www/issues), create [requests](https://github.com/mumble-voip/mumble-www/issues), and create pull requests for implemented change suggestions.
+<!-- is this still the best method to get into contact? -->
```

Comment:
You mean for bug reports and feature/change requests? Yes it is. Except for reporting security related issues in which case it'd be best to report them via mail to `security@mumble.info`

For further questions we should refer to the forums and as a last resort, we could mention `contact@mumble.info`

Reply:
This is not the part that refers to.

If you look at the whole document, you see that this is about contacting you for development over IRC.

Reply:
Ah okay my bad. Yes IRC (#mumble-dev channel on Freenode or via Matrix: https://app.element.io/#/room/!VNUpYnUPdhTAqagvUu:matrix.org) is still the way to go. Additionally creating a PR or issue (or commenting on an existing one) would also work just fine :point_up:

Reply:
Is there some generic link for your matrix channel/server?
I only found links that include web applications, like your link above.
(Sry if I maybe missunderstand something, I don't use matrix)

Reply:
I have not yet found a way to link to IRC channels and the matrix link is the only one that I know.
For everyone that wants to use plain IRC we simply have to tell them to connect to Freenode and then join the #mumble-dev channel

Reply:
For IRC this link was already included in the original file: irc://chat.freenode.net/mumble-dev
(the browser will then ask for an application to open the link)
I suspect that something similar might exist for matrix.

Reply:
It seems it could be this: `#freenode_#mumble-dev:matrix.org`   (copied from server search)

Examples given for normal connections in many matrix clients is: `Type /join #matrix:matrix.org`

Reply:
Idk. Could work, yes.
The link I provided lets you choose which application you want to use to open it though (admittedly it is a bit hidden, but there is a "change" button that lets you choose another app) :shrug:

Reply:
I will add both.